modules/data-entry/src/main/frontend/src/dataHomepage/Questionnaires.jsx

```diff
@@ -51,7 +52,8 @@ function Questionnaires(props) {
   ]
   const actions = [
     DeleteButton,
-    EditButton
+    EditButton,
+    ExportButton
```

Comment:
I think the order in the UI should be Edit, Export, Delete, so the order in the list should be:
```suggestion
    ExportButton,
    EditButton
```